pages-sections/about-us/TeamMemberCard.js

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+import Card from "../../components/Card/Card";
+import CardAvatar from "../../components/Card/CardAvatar";
+import CardBody from "../../components/Card/CardBody";
+import Button from "../../components/CustomButtons/Button";
+import Popover from '@material-ui/core/Popover';
+import Typography from '@material-ui/core/Typography';
+
+const TeamMemberCard = ({ classes, member}) => {
+  const [anchorEl, setAnchorEl] = React.useState(null);
+
+  const handleClick = (event) => {
+    setAnchorEl(event.currentTarget);
+  };
+
+  const handleClose = () => {
+    setAnchorEl(null);
+  };
+
+  const open = Boolean(anchorEl);
+  // const id = open ? 'simple-popover' : undefined;
+  const id = 'popover'
+
+  return (
```

Comment:
This component does not use 'GridItem' so that it will be possible to reuse outside of a Grid.